internal/common/filespec.go

```diff
@@ -251,15 +261,47 @@ func (this *FileSpec) copyPath(baseDir string, destDir string, onlyContents bool
 		destPath = filepath.Join(destDir, filename)
 	}
 
+	paths, err := filepath.Glob(fileSpecPath)
+	if err != nil {
+		return fmt.Errorf("Failed to resolve the path pattern '%s': '%w'.", this.Path, err)
+	}
+
+	if len(paths) == 0 {
+		return fmt.Errorf("No targets found for the path '%s'.", this.Path)
+	}
+
+	if util.IsFile(destPath) && len(paths) > 1 {
+		return fmt.Errorf("Cannot copy multiple targets into the existing file '%s'.", destDir)
+	}
+
+	if !util.PathExists(destPath) && len(paths) > 1 {
```

Comment:
I think it would make sense to merge these
(an outter if for having multiple paths, then error quick if it's a file).

If you did have two, you should do the len check first, since it is quicker.